docs/examples/compose.md

```diff
@@ -0,0 +1,22 @@
+# Compose
+
+## Docker Desktop
+
+To use Docker's Compose tool to build and run a Testcontainers environment in a Docker Desktop Wormhole configuration,
+it is necessary to override Testcontainers Docker host resolution and set the environment variable `TESTCONTAINERS_HOST_OVERRIDE` to `host.docker.internal`.
+Otherwise, Testcontainers cannot access sibling containers like the Resource Reaper Ryuk or other services running on the Docker's host.
```

Comment:
```suggestion
Otherwise, Testcontainers cannot access sibling containers like the Resource Reaper Ryuk or other services running on the Docker host.
```